pep-0696.rst

```diff
@@ -507,6 +529,19 @@ convenient, could have a ``TypeVarLike`` with no default follow a
 This would have also been a breaking change for a small number of cases
 where the code relied on ``Any`` being the implicit default.
 
+Allowing ``TypeVarLike``\ s to be used in function signatures
```

Comment:
Do you mean "TypeVarLike" with a default type? Obviously, "TypeVarLike"s are already supported in function signatures today.